cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/TableSourceConfig.java

```diff
@@ -41,6 +41,7 @@
 
   @Name(Properties.Table.PROPERTY_SCHEMA_ROW_FIELD)
   @Description("The name of the record field that should be used as the row key when writing to the table.")
```

Comment:
I think the description should change to 'Field name that for the Table's row key' ? If this property is not provided, will we create the structured record only from the columns of the row?